packages/whale-api-client/src/api/poolpair.ts

```diff
@@ -45,8 +45,15 @@ export interface PoolPairData {
     reserve: string // BigNumber
     blockCommission: string // BigNumber
   }
+  priceRatio: {
+    'tokenA/tokenB': string // BigNumber | string
+    'tokenB/tokenA': string // BigNumber | string
```

Comment:
```suggestion
    ab: string // BigNumber | string
    ba: string // BigNumber | string
```

so that we can access it naturally without using `[]` for data access consistency across platform